addons/project/tests/test_project_recurrence.py

```diff
@@ -569,12 +570,23 @@ def get_task_and_subtask_counts(domain):
             'project_id': self.project_recurring.id
         })
         domain = [('project_id', '=', self.project_recurring.id)]
+        child_task_1 = self.env['project.task'].create({
+            'name': 'Child task 1',
+        })
+        child_task_2_recurrence = self.env['project.task'].create({
+            'name': 'Child task 2 that have recurrence',
+        })
```

Comment:
You can create in batch 🙂 
```suggestion
        child_task_1, child_task_2_recurrence = self.env['project.task'].create([
            {'name': 'Child task 1'},
            {'name': 'Child task 2 that have recurrence'},
        ])
```